test/featureTestRunner.js

```diff
@@ -121,19 +130,27 @@ function setOptions(load, prologOptions) {
   load.metadata.traceurOptions = traceurOptions;
 }
 
-function featureTest(name, url) {
+function featureTest(name, url, overrideOptions = undefined) {
   test(name, (done) => {
     let baseURL = './';
 
     let prologOptions;
     function translateSynchronous(load) {
       let source = load.source;
+
       // Only top level file can set prologOptions.
```

Comment:
This is actually an old comment. It comes from the case where the main test uses `import`, which means that we hit the `translateSynchronous` for the imported files. I'll update the comment.